test/jdk/java/nio/Buffer/BasicChar.java

```diff
@@ -527,18 +527,6 @@ private static void checkSlice(CharBuffer b, CharBuffer slice) {
 
 
 
-
```

Comment:
These white space changes come from re-generating the test files using the script included in the repo. I've kept the changes since so that the next contributor won't have to remove them again.